### PR TITLE
TRAC-1244: Welt apps doPlugins

### PR DESCRIPTION
**Pull request by mehakraza**

### CODE REVIEW

File: extensions/doPlugins_welt_apps.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
+const s = window.s || window.cmp || {};
```
Comment (HeikoKampe) [inline extensions/doPlugins_welt_apps.js:2]: We do not have the CMP stuff in App context and therefore never a global cmp object.

File: tests/doplugins_welt_apps/welt_apps_page_age_for_checkout.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should store the screen age publication (utag.data.screen_agePublication) in the utag_main cookie if doc type is article', () => {
+        window.utag.data.adobe_docType = 'article';
+
+        s._setPageAgeForCheckout();
+
+        expect(window.utag.loader.SC).toHaveBeenCalledWith('utag_main', { 'pa': window.utag.data.screen_agePublication + ';exp-session' });
```
Comment (HeikoKampe) [inline tests/doplugins_welt_apps/welt_apps_page_age_for_checkout.test.js:28]: Better set window.utag.data.screen_agePublication to a test value before calling the tested function.

**Review by HeikoKampe — APPROVED** [review]
Found two tiny issues :+1:

File: tests/doplugins_welt_apps/welt_apps_page_age_for_checkout.test.js (hunk 2 above)
Comment (mehakraza) [reply tests/doplugins_welt_apps/welt_apps_page_age_for_checkout.test.js:29]: Thanks for pointing out. Updated.

**Pull request merged**